ts/IterableInboxMessageDisplay.tsx

```diff
@@ -107,16 +94,11 @@ const IterableInboxMessageDisplay = ({
          Iterable.savedConfig.urlHandler(event.nativeEvent.data, context)
       }
 
-      returnToInbox()
-   }
-       
-   const openExternalURL = (event: any) => {
-      if (event.url.slice(0, 4) === 'http') {
-         Linking.openURL(event.url)
-         returnToInbox()
-         return false
+      if (URL.slice(0, 4) === 'http') {
+         Linking.openURL(URL)
```

Comment:
Are we not using `Linking.canOpenUrl`?

Reply:
Using canOpenUrl() returns a promise that needs to be processed. I thought using this would be more straight forward and accomplish what we want.

Reply:
Actually we can use canOpenUrl(). I will make the update.

Reply:
Using canOpenUrl() causes issue with the error links. I think we should go with the current approach.